appservice/src/createAppService/AppServicePlanSkuStep.ts

```diff
@@ -67,7 +68,8 @@ export class AppServicePlanSkuStep extends AzureWizardPromptStep<IAppServiceWiza
                 family: 'F',
                 capacity: 1,
                 label: localize('freeLabel', 'Free'),
```

Comment:
Now that we have "Additional Options" as the label, it feels like we shouldn't include B1 and P1v2 in "Additional Options" since they're not really...additional... (they're already in recommended). However, people won't know the "Recommended" "Basic" is "B1" necessarily, so here is my suggestion:

1. Change the Recommended labels to "Free (F1)", "Basic (B1)" and "Premium (P1v2)"
2. Remove B1 and P1v2 from advanced skus since they're already in recommended

And posting a screenshot would be nice if you do that :)